src/main/kotlin/lotto/view/LottoView.kt

```diff
@@ -5,7 +5,7 @@ import lotto.domain.LottoResult
 
 class LottoView {
     fun drawLottos(lottoResult: LottoResult) {
-        lottoResult.lottos.forEach { drawLotto(it) }
+        lottoResult.lottos.lottos.forEach { drawLotto(it) }
```

Comment:
```suggestion
        lottoResult.lottos.lottos.forEach(::drawLotto)
```
이러한 구문을 함수 reference로 넘길 수 있습니다.

lambda로 작성하는 것과 어떤 차이가 있는지 참고할 수 있는 글을 소개해드릴게요 :slightly_smiling_face:
https://proandroiddev.com/kotlin-lambda-vs-method-reference-fdbd175f6845